packages/travix-ui-kit/components/tabs/tab.css

```diff
@@ -1,76 +1,67 @@
 .ui-tab {
+  background-color: var(--tx-generic-color-blank);
+  box-sizing: content-box;
+  color: var(--tx-generic-color-primary-dark);
   cursor: pointer;
-  flex: 1 1 auto;
-  padding: 4px 0 0;
-  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0px; /* flex-basis: Opx is mandatory for IE */
+  height: 72px;
```

Comment:
What if a content of a tab requires more height than 72 px ?
Do you think to add min-height: 72 px instead ?